### ISSUE TITLE
Ablation experiments of new masking strategies pre-training vs. forecast training

### ISSUE THREAD

**Issue by shmh40** [ecmwf/WeatherGenerator#878]
Run experiments to determine if pre-training with recent masking strategies (including causal masking, combination masking) can lead to better performance on forecasting (following fine-tuning) compared to straight forecast training. Focus on obs datasets. Likely will pre-train on Alps with 4 dataset case to analyse this.

**Comment by shmh40** [comment]
@csjfwang could you assign yourself to this issue too please?
